hypervisor/init_comm.go

```diff
@@ -11,8 +11,8 @@ import (
 
 // Message
 type DecodedMessage struct {
-	code    uint32
-	message []byte
+	Code    uint32
+	Message []byte
 }
```

Comment:
There is no big difference to seperate one commit to two,
exec support depends on this change, so they are in the same commit.
there is nothing wrong.